pkg/cmd/tenant/view/view.go

```diff
@@ -1,47 +1,40 @@
 package view
 
 import (
+	"errors"
 	"fmt"
+
 	"github.com/OctopusDeploy/cli/pkg/apiclient"
-	"io"
 
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/OctopusDeploy/cli/pkg/constants"
 	"github.com/OctopusDeploy/cli/pkg/factory"
 	"github.com/OctopusDeploy/cli/pkg/output"
-	"github.com/OctopusDeploy/cli/pkg/usage"
 	"github.com/OctopusDeploy/cli/pkg/util/flag"
-	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
 	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 )
 
 const (
-	FlagWeb = "web"
+	FlagTenant = "tenant"
+	FlagWeb    = "web"
 )
 
 type ViewFlags struct {
-	Web *flag.Flag[bool]
+	Tenant *flag.Flag[string]
+	Web    *flag.Flag[bool]
 }
 
 func NewViewFlags() *ViewFlags {
 	return &ViewFlags{
-		Web: flag.New[bool](FlagWeb, false),
+		Tenant: flag.New[string](FlagTenant, false),
+		Web:    flag.New[bool](FlagWeb, false),
 	}
 }
 
-type ViewOptions struct {
-	Client   *client.Client
-	Host     string
-	out      io.Writer
-	idOrName string
-	flags    *ViewFlags
-}
-
```

Comment:
@domenicsim1 I removed this struct because it seemed simplier without it. Should this be retained for unit test purposes?